plugins/adr/README.md

```diff
@@ -71,16 +71,32 @@ Afterwards, add the following code snippet to use `AdrSearchResultListItem` when
 import { AdrSearchResultListItem } from '@backstage/plugin-adr';
 
 ...
+<SearchType.Accordion
+  name="Result Type"
+  defaultValue="software-catalog"
+  types={[
+    ...
+    {
+      value: 'adrs',
+      name: 'Architecture Decision Records',
+      icon: <DocsIcon />,
+    },
+  ]}
+/>
 
-case 'adr':
+...
+
+case 'adrs':
   return (
     <AdrSearchResultListItem
       key={document.location}
-      result={document}
+      result={document as AdrDocument}
     />
   );
 ```
 
+> Note the `AdrDocument` might not be needed in newer versions https://github.com/backstage/backstage/issues/13723
```

Comment:
Maybe we should mention here that this is not required if you're leveraging the new search results extensions available in v1.11 + : https://backstage.io/docs/features/search/how-to-guides#2-using-an-extension-in-your-backstage-app

Reply:
I'll add this comment, but will not add new docs on past v1.11 for now as I haven't tried it out.